src/components/themed/TransactionRow.tsx

```diff
@@ -50,6 +49,8 @@ const TransactionRowComponent = (props: Props) => {
 
   const { canReplaceByFee = false } = wallet.currencyInfo
 
+  const isSentTransaction = transaction.nativeAmount.startsWith('-') || (eq(transaction.nativeAmount, '0') && transaction.isSend)
+
   const cryptoAmountString = `${isSentTransaction ? '-' : '+'} ${denominationSymbol ? denominationSymbol + ' ' : ''}${cryptoAmount}`
```

Comment:
Using `isSentTransaction` on this line seems like a bug. We should put a - sign or a + sign based on whether the amount is positive or negative, and nothing else.

Reply:
The only time this would be borderline incorrect is if the amount is '0' such as for Algorand token activations which show 0 amount spends on the token tx. At worst it would show "-0" sent which is "kind of" correct.

Reply:
Why the heck does number formatting have anything to do with who signed the transaction!? The point of adding `isSend` was to *fix* bugs like this, not just replace the existing logic. I will approve this, but when QA complains about "-0" transactions, you get to fix it.